pkg/_pub_shared/lib/search/search_form.dart

```diff
@@ -69,6 +70,33 @@ SearchOrder? parseSearchOrder(String? value) {
   return null;
 }
 
+int? parseUpdatedInDays(String value) {
```

Comment:
Feel free to ignore this suggestion from the JS I did years ago, but we have the same regex engine... It's been adopted, just inspired by the regex we used there:
```dart
final timeExp = new RegExp(
  [
    '^((-|\\+))?',
    '((?<years>\\d+)(y|year|years|yr))?',
    '((?<months>\\d+)(months|month|mo))?',
    '((?<weeks>\\d+)(weeks|week|wk|w))?',
    '((?<days>\\d+)(days|day|d))?',
    '((?<hours>\\d+)(hours|hour|hr|h))?',
    '((?<minutes>\\d+)(minutes|minute|min|m))?',
    '((?<seconds>\\d+)(seconds|second|sec|s))?',
    '\$',
  ].join(''),
  caseSensitive: true,
);

Duration parseTime(String duration) {
  // Parse the string
  final match = timeExp.firstMatch(duration);
  if (match == null) {
    throw FormatException('String: "$duration" isn\'t a time expression');
  }
  // Negate if needed
  final neg = match.group(2) == '-' ? -1 : 1;
  // Return parsed values
  return Duration(
    days: (365 * (int.tryParse(match.namedGroup('years') ?? '0') ?? 0) * neg +
        31 * (int.tryParse(match.namedGroup('months') ?? '0') ?? 0) * neg +
        7 * (int.tryParse(match.namedGroup('weeks') ?? '0') ?? 0) * neg +
        (int.tryParse(match.namedGroup('days') ?? '0') ?? 0) * neg),
    hours: int.tryParse(match.namedGroup('hours') ?? '0') ?? 0 * neg,
    minutes: int.tryParse(match.namedGroup('minutes') ?? '0') ?? 0 * neg,
    seconds: int.tryParse(match.namedGroup('seconds') ?? '0') ?? 0 * neg,
  );
}
```

Reply:
updated the parsing with expressions from here, using `m` for months and ignoring minute/seconds level details.

Reply:
PTAL